### DISCUSSION TITLE
GitHub does not track my contributions!

### DISCUSSION THREAD

**Discussion by FaezehMosayyebi** [community/community#121493] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello!
GitHub does not show my contributions.
I've added my email to the git config and my private contribution show is on. 
I send commits to the right remote repo.
The contributions are in one of my repos (not forked).
How can I fix it?

**Comment by parikhitritgithub** [comment]
Sometimes, contributions may not appear immediately. Try making another contribution by creating a new repository, copying it to your VS Code, and pushing it. If your contributions still don't show up on your profile, they may still appear in your activity feed. Check your GitHub profile's recent activity feed to see if your contributions are listed there.

**Reply by FaezehMosayyebi** [reply]
Thank you for your answer.
I merged the branch with my main branch and the related commits appeared in my contribution tracker.
